public class First
{
  public static void main(String args[])
   {
    System.out.println("Exception1");
   try{
      int a=100/0;
   }
   catch(ArithmeticException e)
        {  System.out.println(e);  }

    System.out.println("Exception2");
    try{
       int arr[ ]=new int[5];
       arr[7]=60;
     }
   catch(ArrayIndexOutOfBoundsException e)
      {   System.out.println(e);  }

   System.out.println("Exception3");
  try
    {
      String s=null;  
      System.out.println(s.length());
    }
catch (NullPointerException e)
{   System.out.println(e); }

System.out.println("Exception4");
try
{
  String s="kiran";  
  int i=Integer.parseInt(s);
}
catch (NumberFormatException e)
{    System.out.println(e);  }



System.out.println("Universal Exception");
try{
   int a=100/0;
  }
catch (Exception e)
{   System.out.println(e);  }

finally
{
System.out.println("Univerasal or Generic Exception we can use for other types of exceptions also");
  }
}
}
/* OUTPUT:
Exception1
java.lang.ArithmeticException: / by zero

Exception2
java.lang.ArrayIndexOutOfBoundsException: Index 7 out of bounds for length 5

Exception3
java.lang.NullPointerException: Cannot invoke "String.length()" because "<local1>" is null

Exception4
java.lang.NumberFormatException: For input string: "kiran"
Universal Exception

java.lang.ArithmeticException: / by zero
Univerasal or Generic Exception we can use for other types of exceptions also
*/